terraform/trial_rg/modules/fhir/main.tf

```diff
@@ -2,6 +2,9 @@ resource "random_password" "fhir_sql_password" {
   length = 16
   special = true
   override_special = "_%@"
+  min_upper        = 1
```

Comment:
is this because the sporadic failure?

Reply:
correct, it happened to me 2-3 times so I looked at the SQL policy and matched it.